### ISSUE TITLE
potential router additions

### ISSUE THREAD

**Issue by jawz101** [gwarser/filter-lists#17]
I notice these because I occasionally look at the daily [Cisco Umbrella Top 1 Million](http://s3-us-west-1.amazonaws.com/umbrella-static/index.html) DNS lookup list a bit to see what's going on in the word and see some of the routers that you've listed in your rules.  I didn't know if any of these might be additional suggestions for the router list.  Basically, I just searched for anything in the csv that started with google.com. since it is already popular and notice that there are several proxy-ish solutions (either private or commercial) that like to handle traffic for people.  I don't know why but whatever.

Ones that I notice often are the bbrouter, asus, belkin, dlink, totolink, tendawifi, huaweimobilewifi, zyxel, & securly if my memory serves anything useful.

```
google.com.1.14ea4o9r8e77k.1.prx.uswest2.v1api.securly.com
google.com.1.1lrj30omrtwhx.1.prx.useast.v1api.securly.com
google.com.ado-piso-wifi.com
google.com.apple.com
google.com.asus
google.com.bbrouter
google.com.beeline
google.com.belkin
google.com.blinkap
google.com.broadband
google.com.bwrouter
google.com.cantv.net
google.com.claro.local
google.com.corp.costco.com
google.com.cpe.gye.telmex.ec.intranet
google.com.cpe.tdp.com
google.com.dhcp
google.com.dlink
google.com.dlinkrouter
google.com.dnsbl7.mailshell.net
google.com.dob.sibl.support-intelligence.net
google.com.domain.name
google.com.ec2.internal
google.com.freewave.at
google.com.fresh.fmb.la
google.com.fritz.box
google.com.gateway.huawei.net
google.com.guest.local
google.com.guest.store
google.com.hitronhub.home
google.com.hm.com
google.com.home
google.com.home.gateway
google.com.home.sercomm
google.com.hsd1.fl.comcast.net
google.com.huawei.net
google.com.intern
google.com.itotolink.net
google.com.local
google.com.local.tld
google.com.localdomain
google.com.localhost
google.com.localnet
google.com.lookup.dkimwl.org
google.com.maxprint
google.com.mcas.ms
google.com.meuintelbras.local
google.com.microsoft.ir.microsoft.biz.inrtenet-miazan-ir.me
google.com.multi.uribl.com
google.com.mygowifi.com
google.com.myncscloud.com
google.com.mynet
google.com.nebo.edu
google.com.onion
google.com.portal.pisofi.com
google.com.realtek
google.com.ref.ualibrary.org
google.com.repeater.com
google.com.rga.ip
google.com.shoprite.net
google.com.speedport.ip
google.com.station
google.com.t.hu
google.com.tendawifi.com
google.com.tl-wa850re
google.com.tl-wa855re
google.com.totolink
google.com.unch.unc.edu
google.com.uribl.spameatingmonkey.net
google.com.us-east-1.ec2-utilities.amazonaws.com
google.com.webhosting-apple.credit
google.com.webhostingtalk.ga
google.com.wework.com
google.com.wireless-n
google.com.workgroup
google.com.www.everestwifi.com
google.com.www.huaweimobilewifi.com
google.com.www.mtlinkwifi.com
google.com.www.nexxtwifi.com
google.com.www.nexxtwifi.local
google.com.www.qpcomwifi.com
google.com.www.roteador.multilaser.com
google.com.www.tendawifi.com
google.com.zte.com.cn
google.com.zuku.co.ke
google.com.zyxel-usg
google.com.zyxel.com
```

**Comment by gwarser** [comment]
Why they start with `google.com`?

**Comment by jawz101** [comment]
I just saw you have rules such as

||fritz.box^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||fritz.nas^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||fritz.repeater^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||giga.cube^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||huaweimobilewifi.com^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||myfritz.box^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||pi.hole^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||repeater.asus.com^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||router.asus.com^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||speedport.ip^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local
||tplinkwifi.net^$third-party,domain=~localhost|~127.0.0.1|~[::1]|~0.0.0.0|~[::]|~local

which I thought meant you were doing something with these sorts of router products.

**Comment by gwarser** [comment]
Yes, but why will Asus use `google.com.asus` to access their routers?

**Comment by jawz101** [comment]
I have no idea :P  I was more curious when I saw the tplink, fritzbox, huaweimobilewifi names I was reminded of such domain lookups I see on that Cisco name lookup list and wondered if there were other potential boxes like that.  Really, this issue can be closed.  I just figured I'd reach out with the info.

**Comment by jawz101** [comment]
... that Cisco list is full of interesting DNS lookups.  Sometimes I see domain lookups that look like they escaped an internal network and end up being looked up publicly- which I found interesting.  I wouldn't think a private company would want their internal network names leaked to a public domain resolver.

**Comment by gwarser** [comment]
Maybe some of these with `google.com` dropped can be added as router addresses. I will look into this later.
